src/client/components/MessageForm/SendMessageForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import './SendMessageForm.css';
+
+function SendMessageForm() {
+  const [input, setInput] = useState('');
+  const data = {
+    channel_id: null,
+    user_id: null,
+    message: input,
+  };
+  const handleSubmit = () => {
+    fetch('/api/messages', {
+      method: 'post',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    })
+      .then((res) => res.json())
+      .then((messages) => console.log(messages));
+  };
+
+  return (
+    <footer>
```

Comment:
This is not a footer. I would just do it as the form. So the form is the wrapping element

Reply:
yes i used as a form.Thank you.